modules/machine-config-pin-preload-images.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * machine_configuration/machine-config-pin-preload-images-about.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machine-config-pin-preload-images_{context}"]
+= Pinning and preloading images
```

Comment:
Here and throughout this module, should the order be "Pinning and preloading images" or "Preloading and pinning images"?